packages/block-editor-tools/src/components/selector/index.jsx

```diff
@@ -52,6 +53,9 @@ const Selector = ({
   // Create ref.
   const ref = useRef();
 
+  // Memoize subType string.
+  const selectedSubTypes = useMemo(() => (subTypes.length > 0 ? subTypes.join(',') : 'any'), [subTypes]);
```

Comment:
This has been memoized because when the `multiple` prop is passed, it would cause the returned post list to multiply with duplicate entries.